react_ws_src/src/helpers/computer_moves.js

```diff
@@ -0,0 +1,168 @@
+const BOARD_SIZE = 9;
+const CELL_PREFIX = 'c';
+const PLAYER = 'x';
+const COMPUTER = 'o';
+
+/**
+ * Finds the best move for the computer using the minimax algorithm.
+ * It converts the move index into the cell id used within the UI.
+ *
+ * NOTE: ideally we wouldn't have to convert between the cell id and the index since that is a UI concern.
+ * Nevertheless, I felt it was better to encapsulate everything into 1 function. If I had more time I'd refactor
+ * the component a bit to create clearer separation of concerns between the move logic and the presentation.
+ */
```

Comment:
This is the main entry point. I prefer named exports over `defaults` because it means you retain the same function name everywhere.

As the `note` above suggests, ideally we would not be passing in data that is coupled to the UI. However, it was cleaner to handle the conversion from `cell` to `index` and back here, rather than in the component, at least for now. It allowed me to test it more thoroughly and be more confident it would work.